### PR TITLE
Pig Latin: Remove use of "constant clusters"

### PR DESCRIPTION
**Pull request by iHiD**
Context: https://forum.exercism.org/t/pig-latin-ch-in-chair-is-not-consonant-cluster/6285

> maybe try to use those confusing terms as little as possible until fully understand them. the fact that there are different definitions for consonant clusters and consonant digraphs on different websites further proves that there is no reason to continue using them.

### CODE REVIEW

**Review by IsaacG — APPROVED** [review]

**Review by BethanyG — APPROVED** [review]
This reads much better. _Many thanks!_

**Comment by kotp** [comment]
I guess I do not count as a "third" for approval. ;(

**Comment by BethanyG** [comment]
@kotp - I think it's more that even with three maintainers, it takes a `maintainers-admin` before merging can happen.

**Review by ErikSchierboom — APPROVED** [review]
Done

**Pull request merged**
